cl/opinion_page/views.py

```diff
@@ -1087,8 +1111,13 @@ def citation_homepage(request: HttpRequest) -> HttpResponse:
         if form.is_valid():
             # Redirect to the page as a GET instead of a POST
             cd = form.cleaned_data
+            kwargs = {"reporter": cd["reporter"]}
+            if cd["volume"]:
+                kwargs["volume"] = cd["volume"]
+            if cd["page"]:
+                kwargs["page"] = cd["page"]
```

Comment:
Do you need this? I'm just imagining here, but if you just pass through `cd` like before, aren't volume and page None, and doesn't that work fine?

Reply:
I originally tried it with just passing in 'cd' but that caused the reverse() function to blow up because it didn't find a match in urls.py that it liked.

Reply:
Got it, thanks.